docs/Change_Log.md

```diff
@@ -3,6 +3,12 @@
 >danger
 >API and Gateway versions below v6 will be discontinued on October 16, 2017, after which they will be non-functioning.
 
+## July 31, 2017 - Breaking Change
+
+After today, we are removing the existing concept of a default channel from Discord. We saw a use case in many servers where the previously-default #general channel was being repurposed as an announcement-only, non-writable channel for new members. In practice, server owners were using bots to clear the entire message history of the default channel in order to change its purpose. Now that channel can simply be deleted and re-created with the desired permissions.
+
+From now on, new servers will no longer have a default channel with the same snowflake id as its guild. Existing guilds are unaffected. This means that your library can no longer assume that channel as a default. The "default" channel for a given user is now the channel with the highest position that their permissions allow them to see.
```

Comment:
Sorry that needs clarification. It means that existing guilds will not have their `#general` channel id changed. It will, however, no longer be the default channel.